docker/docker-compose-build.yml

```
@@ -46,3 +46,15 @@ services:
       - bind9
       - elasticmq
       - dynamodb
+
+  portal:
+    image: "vinyldns/portal:0.1"
+    ports:
+      - "9001:9001"
+    container_name: "vinyldns-portal"
+    volumes:
+      - ./portal/application.conf:/opt/docker/conf/application.conf
+      - ./portal/application.ini:/opt/docker/conf/application.ini
+      # - ./portal/trustStore.jks:/opt/docker/conf/trustStore.jks  # uncomment if using a custom trustStore
```

I don't think we need this.  The "how to add a trust store" should be in the "using the portal docker image"

---

@pauljamescleary Should these be taken out completely or can they be mounted by default since it will fail silently if the files aren't there